bindata/network/ovn-kubernetes/common/ipsec.yaml

```diff
@@ -207,13 +207,13 @@ spec:
           # Start the pluto IKE daemon
           /usr/libexec/ipsec/pluto --leak-detective --config /etc/ipsec.conf --logfile /var/log/openvswitch/libreswan.log
 
-          # Environment variables are for workaround for https://mail.openvswitch.org/pipermail/ovs-dev/2020-October/375734.html
-          # We now start ovs-monitor-ipsec which will monitor for changes in the ovs
+          # Start ovs-monitor-ipsec which will monitor for changes in the ovs
```

Comment:
Hi @andreaskaris,
thank you for submitting this fix.
i see that you are trying to save time by just executing the script in the foreground, i agree with you that we don't really care about the output of readinessProbe, livenessProbe executed commands but sometimes these commands return code can be  useful and can help to indicate an issue in IPsec,  for example:
if you have an issue with Pluto or it's not running for some reason this change will keep reporting that the pod is running fine but nothing really works, in the other hand the ipsec status command will returns non-zero code which will then make the pod failed to create.

please see this output example:
with your change enabling ipsec when pluto is not running will run fine:
`python3 /usr/share/openvswitch/scripts/ovs-monitor-ipsec --pidfile=/var/run/openvswitch/ovs-monitor-ipsec.pid --ike-daemon=libreswan --log-file --monitor unix:/var/run/openvswitch/db.sock 
2022-10-31T12:03:16Z |  0  | ovs-monitor-ipsec | INFO | Restarting IKE daemon
2022-10-31T12:03:16Z |  1  | ovs-monitor-ipsec | INFO | Restarting LibreSwan
Redirecting to: systemctl restart ipsec.service
Job for ipsec.service failed because the control process exited with error code.
See "systemctl status ipsec.service" and "journalctl -xe" for details.
2022-10-31T12:03:16Z |  3  | reconnect | INFO | unix:/var/run/openvswitch/db.sock: connecting...
2022-10-31T12:03:16Z |  6  | reconnect | INFO | unix:/var/run/openvswitch/db.sock: connected


keeps running .....
`

with the current implantation the  ipsec status command will return with code 33:
`ipsec status
whack: Pluto is not running (no "/run/pluto/pluto.ctl")
[root@localhost ~]# echo $?
33
`
what about maybe we keep the current implantation and remove the ovs-appctl -t ovs-monitor-ipsec ipsec/status from the probs and keep the ipsec status command, that will save us more time since we do the same command twice now.

Reply:
Also, our plan is to add a more complex probe / another container that actually runs a check that's a bit smarter and that will taint the node if not all tunnels are up. The idea is to avoid scheduling workloads to nodes where the ipsec pod is causing issues. That said, that would be in a follow-up commit

Reply:
> 

yes, you are right, we don't need the readinessProbe since we don't get traffic.

Reply:
@mohammadheib  Does this look better to you, now? (I'd still like to run the command itself in the foreground instead of the `sleep`, so I left it that way)